### PR TITLE
Fix/Free-Form Object issue

### PR DESCRIPTION
**Pull request by DevAgani**
API Specs with properties defined with the type `object` a.k.a free form objects always have child properties with types defined clearly. 

In the event that these child properties do not have types defined explicitly, it is expected that the definition of the parent will have a property called `additionalProperties` [read more](https://swagger.io/docs/specification/data-models/dictionaries/).

This PR seeks to intervene the scenario where by the child properties don't have types and no `additionalProperties` exists.

### CODE REVIEW

File: boat-scaffold/src/main/java/org/openapitools/codegen/languages/BoatSwift5Codegen.java [hunk 1]
```diff
@@ -62,6 +69,49 @@ public String getTypeDeclaration(Schema p) {
...
+        if (codegenModel.description != null) {
+            codegenModel.imports.add("ApiModel");
+        }
+
+        fixAllFreeFormObject(codegenModel);
+
+        return codegenModel;
+    }
+
+    private void fixAllFreeFormObject(CodegenModel codegenModel) {
```
Comment (burgbart) [inline boat-scaffold/src/main/java/org/openapitools/codegen/languages/BoatSwift5Codegen.java:85]: Would be good to add documentation to this method why this is needed. e.g. is it a workaround for a bug in the OpenApiClientGenerator, is it simply for backwards compatibility, what is the required behaviour vs current behaviour etc.

File: boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java [hunk 2]
```diff
@@ -173,6 +167,26 @@ public void testPodAuthors() throws Exception {
         assertEquals(podAuthors, openAPIDevs);
     }
 
+    @Test
+    public void testFromModel() {
```
Comment (burgbart) [inline boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java:171]: There are no assertions in this test, what is tested exactly?

Comment (DevAgani) [reply boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java:171]: I forgot to add an assertion, i'll add a simple assertion.

File: boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java [hunk 3]
```diff
@@ -173,6 +174,108 @@ public void testPodAuthors() throws Exception {
...
```
Comment (bdomok) [inline boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java:178]: Would be nice to include a schema that meets the criteria for the above fix (free form object without additional properties).

Comment (DevAgani) [reply boat-scaffold/src/test/java/org/openapitools/codegen/languages/BoatSwift5CodegenTests.java:178]: I've added schemas that meet the criteria.

**Review by burgbart — APPROVED** [review]

**Pull request merged**